steps/zoo.cfg

```diff
@@ -9,7 +9,7 @@ syncLimit=5
 # the directory where the snapshot is stored.
 # do not use /tmp for storage, /tmp here is just 
 # example sakes.
-dataDir=/rambuild/ut_zookeeper
+dataDir=./data
```

Comment:
no unrelated changes

Reply:
This change is due to the macos root permission, which caused the Error: `mkdir: /rambuild/ut_zookeeper: Read-only file system`. Therefore, we change the absolute path to relative path, which works for both Linux and Mac.